canvases/dark-editor-vue/src/components/ResponseChat.vue

```diff
@@ -118,7 +118,8 @@ const executeCode = async (index: number) => {
     })
 
     if (!response.ok) {
-      throw new Error('Error in parsing the expr and serializing it as JSON')
+      let error = await response.text()
```

Comment:
While this isn't helping right now, it may come in handy for detecting other errors